src/HttSystematics_MSSMvsSMRun2.cc

```diff
@@ -554,7 +904,7 @@ void AddMSSMvsSMRun2Systematics(CombineHarvester &cb, bool jetfakes, bool embedd
           cb.cp()
               .channel({"tt"})
               .process({"EMB"})
-              .AddSyst(cb, "CMS_eff_xtrigger_t_tt_dm"+tauTriggerbin+"_highpT_$ERA", "shape", SystMap<>::init(0.5));
+              .AddSyst(cb, "CMS_eff_xtrigger_t_emb_tt_dm"+tauTriggerbin+"_highpT_$ERA", "shape", SystMap<>::init(0.5));
```

Comment:
The naming of the uncertainties should not be changed here as it is introduced for the correlation of the uncertainties acting on the MC and Embedded samples. If we do not want to correlate the uncertainties we would also need to update the magnitude of the uncertainty acting on the embedded shapes only.

Reply:
Apologies, I must have forgot to change this back. I changed this originally to work with what we had in our datacards but have since updated the datacards to work with correlated and uncorrelated embedding uncertainties. So please ignore the update to this line.